scripts/clan.py

```
@@ -831,6 +821,20 @@ def load_clan_json(self):
         )
         get_current_season()
 
+        for cat in Cat.all_cats.values():
+            try:
+                # initialization of thoughts
+                cat.thoughts()
+            except Exception as e:
```

Too broad an exception, can we be more specific what we're expecting to handle here?

---

I just copied it over from earlier in the Clan load process.

---

Since this PR is more than a plain refactor, this should be fixed to catch specific errors or let the error propagate to a place where it can be handled sensibly.